### PR TITLE
Fixed numerous grammatical/spelling/consistency errors

### PR DESCRIPTION
**Pull request by Alceris**
In objects/avaligeneric, at least.

### CODE REVIEW

File: objects/avaligeneric/kitchen/avalikitchencupboard.object [hunk 1]
```diff
@@ -14,11 +14,11 @@
 
   "apexDescription" : "This cupboard is remarkably light yet sturdy.",
   "avianDescription" : "This'd make a fantastic trophy cabinet.",
-  "floranDescription" : "What good is a box to hide things in, if you can see through it?",
+  "floranDescription" : "What good isss a box to hide things in, if you can sssee through it?",
   "glitchDescription" : "Bored. Once you have seen one cupboard you have seen them all.",
-  "humanDescription" : "It's a cupboard, could probably get something similar at OKEA",
+  "humanDescription" : "It's a cupboard, could probably get something similar at IKEA.", // OKEA?
```
Comment (Fevix) [inline objects/avaligeneric/kitchen/avalikitchencupboard.object]: Okea was a Starbound mod, now known as "Pixel Goods Store" which was basically Ikea.

File: objects/avaligeneric/kitchen/avalikitchenfridge.object [hunk 2]
```diff
@@ -12,13 +12,13 @@
   "shortdescription" : "Refrigeration Cabinet",
   "race" : "generic",
 
-  "apexDescription" : "Looking at the controls, I suspect this doubles as storage too",
-  "avianDescription" : "I think it's warmer inside this thing than outside",
-  "floranDescription" : "Cold and dark box, florans hates it.",
+  "apexDescription" : "Looking at the controls, I suspect this doubles as storage too.",
+  "avianDescription" : "I think it's warmer inside this thing than outside.",
+  "floranDescription" : "Cold and dark box, floran hates it.",
```
Comment (Fevix) [inline objects/avaligeneric/kitchen/avalikitchenfridge.object]: no "hatesss"?

File: objects/avaligeneric/kitchen/avalikitchenstove.object [hunk 3]
```diff
@@ -12,13 +12,13 @@
...
+  "apexDescription" : "Interesting, the gas rings on this pump oxygen, not methane.",
   "avianDescription" : "It's got a way to properly cook almost anything here!",
-  "floranDescription" : "Floran put wood in box, then heats til it bursts into flame. Campfire for stupids.",
-  "glitchDescription" : "Dubious. I doubt this device gets hot enough to properly melt microcheese until rust-brown",
+  "floranDescription" : "Floran put wood in box, then heats 'till it bursssts into flame. Campfire for sssstupidss.",
+  "glitchDescription" : "Dubious. I doubt this device gets hot enough to properly melt microcheese until rust-brown.",
   "humanDescription" : "I should probably read the instructions before trying to dry my cat in this thing.",
   "hylotlDescription" : "I'm beginning to suspect that for all their flaws, the Avali may have promise as culinarians.",
-  "avaliDescription" : "We breathe methane, our stoves have to pump oxygen which then burns with the methane in the air",
+  "avaliDescription" : "We breathe methane, our stoves have to pump oxygen which then burns with the methane in the air.", // Isn't it hydrogen that they breathe?
```
Comment (Fevix) [inline objects/avaligeneric/kitchen/avalikitchenstove.object]: IDK, and IDGAF, honestly, Ryuu was trying too hard to be 'different'. It doesn't matter what they breathe.

File: objects/avaligeneric/lab/avalisample1.object [hunk 4]
```diff
@@ -9,13 +9,13 @@
...
-  "humanDescription" : "Looks more like a gurney than a table. Why is Avali stuff always so, functional?",
-  "hylotlDescription" : "I wouldn't keep a moth-eaten book on this unit.",
-  "avaliDescription" : "It's a shelf... or maybe a table. Honestly it's just something to place your shinies on.", 
+  "apexDescription" : "This is nothing like the standardised desks the Miniknog supply us with. (wait, what)",
+  "avianDescription" : "Is it a shelf, is a table, who can judge? (wait, what)",
+  "floranDescription" : "This bed is ssssssstupid, it's hard as a table top. (wait, what)",
+  "glitchDescription" : "Observation. This self is grossly over-engineered (wait, what)",
+  "humanDescription" : "Looks more like a gurney than a table. Why is Avali stuff always so, functional? (wait, what)",
+  "hylotlDescription" : "I wouldn't keep a moth-eaten book on this unit. (wait, what)",
+  "avaliDescription" : "It's a shelf... or maybe a table. Honestly it's just something to place your shinies on. (wait, what)",
```
Comment (Fevix) [inline objects/avaligeneric/lab/avalisample1.object]: This one needs a full rewrite it seems.

File: objects/avaligeneric/racks/hammer/unused/avalitier9hammerdisplay.object [hunk 5]
```diff
@@ -9,7 +9,7 @@
   "shortdescription" : "Hammer Rack",
   "race" : "avali",
 
-  "apexDescription" : "A lance is an odd choice of hunting weapon for such small creatures. How do they use them?",
+  "apexDescription" : "A lance is an odd choice of hunting weapon for such small creatures. How do they use them? (yuwot)",
```
Comment (Fevix) [inline objects/avaligeneric/racks/hammer/unused/avalitier9hammerdisplay.object]: Did you really have to add in so many uncommented comments?

**Comment by Fevix** [comment]
Remove the uncommented comments and I'll go ahead and merge.

**Comment by Alceris** [comment]
I'm kinda new to Github, this is my first time even making a pull request, so I'm not entirely sure I know what I'm doing.

**Comment by Fevix** [comment]
The pull request itself is fine. I'm talking primarily about the stuff you put in parentheses that are also inside the quotations. This will show up ingame, such as when an Apex investigates an Avali hammer rack.

You got comments correct on some, using the // to tell Starbound to ignore it, but on others (Such as the specimin jar and aforementioned hammer racks) you have it inside the quotation marks, which tells Starbound to show it ingame.

**Comment by Alceris** [comment]
Yeah, I know, I did that intentionally, just to be silly, since the lines already make no sense. Anyway, I did change stuff some more, do I just close this and make another, more up-to-date pull request?

**Pull request merged**